mlir/include/mlir/Dialect/Rock/IR/RockAttrDefs.td

```diff
@@ -243,11 +247,14 @@ def Rock_XdlopsGemmParamsAttr : Rock_Attr<"XdlopsGemmParams", [RockTuningParamAt
 
   let extraClassDeclaration = [{
     std::string getPerfConfigStr(){
-      const Twine paramsConcat = Twine(getMPerBlock()) + ","
+      const Twine paramsConcat = 
+          Twine(getMPerBlock()) + ","
         + Twine(getNPerBlock()) + ","
         + Twine(getKPerBlock()) + ","
+        + Twine(getKpack()) + ","
```

Comment:
Note: I _think_ this is in the wrong order, and kPack comes after NPerWave? At least going off InitParamsXDL::visit()?

Reply:
You're right. I was confused with last but two but the last two was 1,1